packages/atomic/src/components/commerce/product-template-components/atomic-product-excerpt/e2e/atomic-product-excerpt.e2e.ts

```diff
@@ -0,0 +1,116 @@
+import {test, expect} from './fixture';
```

Comment:
The comments I made above about testing prop validation errors apply here as well.